go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/iver-wharf/wharf-api
 go 1.16
 
 require (
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
```

Comment:
Find it funny how this dependency comes and goes from PR to PR. I have yet to figure out why it is added sometimes and why it sometimes is removed when I run `go mod tidy`, but it seems consistent no matter how many times I run `go mod tidy` in sequence, but a day later and it works the opposite way.

So weird.

Reply:
I have found myself thinking the same thing on numerous occasions...